src/components/sbb-navigation-section/sbb-navigation-section.tsx

```diff
@@ -163,6 +166,33 @@ export class SbbNavigationSection implements ComponentInterface {
     this._triggerElement.addEventListener('click', () => this.open(), {
       signal: this._navigationSectionController.signal,
     });
+    this._element.addEventListener(
+      'keydown',
+      (event) => {
+        if (event.key !== 'Tab') {
+          return;
+        }
+
+        // Dynamically get first and last focusable element, as this might have changed since opening overlay
+        const elementChildren: HTMLElement[] = Array.from(
+          this._element.shadowRoot.querySelectorAll('*')
```

Comment:
Does IS_FOCUSABLE_QUERY not work here?

Reply:
Does this consider slotted elements?

Reply:
as isBreakpoint('zero', 'large') is called twice, I think we should use a common method

Reply:
The `getFocusableElements` method takes care of everything, checking the IS_FOCUSABLE_QUERY and taking in consideration slotted elements.

Reply:
@dauriamarco Can you create a card for that?